protocol/0057-TRAN-transfers.md

```diff
@@ -275,3 +275,18 @@ A user's recurring transfer to a reward account does not occur if there are no p
   - [ ] In the first epoch no trading occurs and nothing is transferred to the reward account at the end of the epoch
   - [ ] In the second epoch, 2 * 10^6 trading occurs, and at the end of the epoch the transfer to the reward account occurs
   - [ ] At the end of the third epoch, no transfer occurs
+
+If the network parameter <transfer.minTransferQuantumMultiple> is modified, this modification is communicated to the frontend clients and applied there appropriately (displayed/too small transfers rejected). (<a name="0058-TRAN-058" href="#0058-TRAN-58">0058-TRAN-058</a>)
```

Comment:
Im happy we can test these ACs, the statement about modification is communicated to the frontend clients - Should we see a trx rejection message returned back from wallet/core/tx_error event to convey the error. Also, should be have a FE scenario to cover ensuring transfer rejections are conveyed correctly on the FE?

Reply:
I'd argue this is badly worded. The value of the network parameter is visible to FE via the usual APIs - they can deal with it as they do with all the other things. 

We may wish to have "push" notifications for FE for various things where it would be equally important - we don't. It would be a completely separate discussion IMO.  

I suggest we just delete this AC.